packit/local_project.py

```diff
@@ -121,16 +121,17 @@ def __init__(
         if refresh:
             self.refresh_the_arguments()
 
-        # p-s gives us both, commit hash for a PR and PR ID as well
-        # since we want to have 'pr123' in the release field, let's check out
-        # the PR itself, so if both are specified, PR ID > ref
-        if pr_id:
-            self.checkout_pr(pr_id)
-            if merge_pr:
-                self.merge_pr(pr_id, target_branch)
-            self.checkout_as_pr_branch(pr_id)
-        elif ref:
-            self.checkout_ref(ref)
+        if self.git_repo:
+            # p-s gives us both, commit hash for a PR and PR ID as well
+            # since we want to have 'pr123' in the release field, let's check out
+            # the PR itself, so if both are specified, PR ID > ref
+            if pr_id:
+                self.checkout_pr(pr_id)
+                if merge_pr:
+                    self.merge_pr(pr_id, target_branch)
+                self.checkout_as_pr_branch(pr_id)
+            elif ref:
+                self.checkout_ref(ref)
```

Comment:
```suggestion
        # skip checkouts if the git repo is not present
        if not self.git_repo:
            return

        # p-s gives us both, commit hash for a PR and PR ID as well
        # since we want to have 'pr123' in the release field, let's check out
        # the PR itself, so if both are specified, PR ID > ref
        if pr_id:
            self.checkout_pr(pr_id)
            if merge_pr:
                self.merge_pr(pr_id, target_branch)
            self.checkout_as_pr_branch(pr_id)
        elif ref:
            self.checkout_ref(ref)
```

Reply:
PTAL